### PR TITLE
issue fixed #20221 add your review text is not show uniformly in Mobi…

### LINKED ISSUE #20221: add your review text is not show uniformly in Mobile view
###Preconditions (*)

    Magento 2.3.0 with sample data
    Php Version 7.2
    Device Size 360  X  640

###Steps to reproduce (*)
    
   Chrome Browser
   Mobile View
   Home Page
   Product listing page 

###Expected result (*)

add your review text should be shown in one line 

###Actual result (*)

add your review text is not shown uniformly

0sf1x9n png png image 452 x 520 pixels

### PR DESCRIPTION
**Pull request by cedarvinda**
issue fixed #20221 add your review text is not show uniformly in Mobile view

### Manual testing scenarios (*)

1. Chrome Browser
2. Mobile View
3. Home Page
4. Product listing page

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @cedarvinda. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/design/frontend/Magento/luma/Magento_Review/web/css/source/_module.less [hunk 1]
```diff
@@ -393,6 +393,21 @@
     }
 }
 
+.media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__m) {
+    .product-reviews-summary{
+        .reviews-actions {
+            a{
+                display: block;
```
Comment (VladimirZaets) [inline app/design/frontend/Magento/luma/Magento_Review/web/css/source/_module.less:400]: Hi @cedarvinda. Thanks for collaboration. Please add space after `a` declaration.

Comment (cedarvinda) [reply app/design/frontend/Magento/luma/Magento_Review/web/css/source/_module.less:400]: @VladimirZaets ok, Thanks

**Comment by cedarvinda** [comment]
@VladimirZaets _module.less file updated as per your instruction.

**Comment by VladimirZaets** [comment]
@cedarvinda thanks

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-3904](https://jira.corp.magento.com/browse/ENGCOM-3904) has been created to process this Pull Request

**Comment by VladimirZaets** [comment]
Hi @cedarvinda. Thanks for collaboration. 
This issue was fixed in scope of this [PR](https://github.com/magento/magento2/pull/20224)

**Pull request closed without merging**